code/modules/projectiles/guns/ballistic/bow.dm

```diff
@@ -0,0 +1,102 @@
+/obj/item/gun/ballistic/bow
+	name = "bow"
+	desc = "A sturdy bow made out of wood and reinforced with iron."
+	icon_state = "bow_unloaded"
+	item_state = "bow"
+	var/icon_state_loaded = "bow_loaded"
+	var/icon_state_firing = "bow_firing"
+	var/item_state_loaded = "bow"
+	var/item_state_firing = "bow"
```

Comment:
Why are these the same? Why even have either one if the item_state never changes?

Reply:
Item state changes,  it does on the hand!  When you use the bow it changrs the sprite to bow_firing and etc,  later i will make a gif how it works!

Reply:
var/item_state_loaded = "bow"
	var/item_state_firing = "bow"
it doesn't change.